components/atoms/ActionButton.js

```diff
@@ -0,0 +1,192 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import { useEffect } from 'react'
+
+/**
+ * Button component
+ */
+export function ActionButton(props) {
+  //Styling for buttons and links
+  const basicStyle =
+    'rounded-sm focus:ring-1 focus:ring-black focus:ring-offset-2'
+  const defaultStyle =
+    'py-2 px-4 bg-custom-blue-blue text-white border border-custom-blue-blue active:bg-custom-blue-dark hover:bg-custom-blue-light'
```

Comment:
We don't have all of these colours defined in Tailwind

Reply:
i agree we can remove the unsued colors and styles

Reply:
until the colours are resolved, just pick a close colour that is represented by tailwind like blue-200 or something like that. we can refactor them later.

Reply:
ActionButton is not used yet on any page, we should wait until we decide on the colours to start deleting code that we might need later